interface-impl-amf/src/main/java/org/mule/amf/impl/validation/OASArrayValidationStrategy.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.amf.impl.validation;
+
+import amf.client.model.domain.AnyShape;
+import amf.client.model.domain.ArrayShape;
+import amf.client.validate.PayloadValidator;
+import amf.client.validate.ValidationReport;
+import com.google.common.base.Joiner;
+import org.json.simple.JSONValue;
+import org.mule.amf.impl.exceptions.ParserException;
+import org.mule.amf.impl.util.LazyValue;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.mule.amf.impl.util.AMFUtils.APPLICATION_JSON;
+import static org.mule.amf.impl.util.AMFUtils.needsQuotes;
+import static org.mule.apikit.ParserUtils.quoteValue;
+
+public class OASArrayValidationStrategy implements ParameterValidationStrategy {
+
+  AnyShape arrayShape;
+  JsonParameterValidationStrategy facet;
```

Comment:
I know that using  a **JsonParameterValidationStrategy** only to use the quote if needed method is an overkill but it wasn't easy to extract the logic. Adding this as remaining work.